cyclonedx-bom/src/external_models/normalized_string.rs

```diff
@@ -45,9 +46,15 @@ impl NormalizedString {
     }
 }
 
-impl ToString for NormalizedString {
-    fn to_string(&self) -> String {
-        self.0.clone()
+impl AsRef<str> for NormalizedString {
```

Comment:
Is there a good reason to implement `AsRef` instead of [`Deref`](https://doc.rust-lang.org/stable/std/ops/trait.Deref.html)?

Reply:
No. Maybe we should have both. `String` also implements `AsRef<str>`, so you can have `fn foo(s: impl AsRef<str>)`.